tests/deprecated_api/test_remove_1-4.py

```diff
@@ -66,3 +67,9 @@ def test_v1_4_0_deprecated_trainer_attributes():
 
     trainer.use_horovod = True
     assert trainer.use_horovod
+
+
+def test_v1_4_0_deprecated_metrics():
```

Comment:
```suggestion
def test_v1_4_0_deprecated_stat_scores_multiple_classes():
```

Reply:
So this naming is taken form 1.3 where they test multiple depreciated metrics - should there be a separate function for each depreciated metric in 1.4?

Reply:
yeah, there is a possibility of deprecating more metrics in the future. ignore this then :)